org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -535,6 +535,8 @@ else if (DisplayType.isList(AD_Reference_ID)
 					}
 					// 	TableName.ColumnName,
 					sqlSELECT.append(lookupSQL).append(" AS ").append(ColumnName).append(",");
+					if (!IsGroupFunction)
+						groupByColumns.add(ColumnName); 
```

Comment:
2 difference from the "AD_Reference_ID == DisplayType.Table" block:

1. It is groupByColumns.add(lookupSQL) at Table block, not groupByColumns.add(ColumnName)
2. No IsGroupFunction check

Please check the inconsistency, thanks.